src/Microsoft.OData.Core/ODataObjectModelExtensions.cs

```diff
@@ -34,6 +34,17 @@ public static void SetSerializationInfo(this ODataResourceBase resource, ODataRe
             resource.SerializationInfo = serializationInfo;
         }
 
+        /// <summary>
+        /// Provide additional serialization information to the <see cref="ODataWriter"/> for <paramref name="resource"/>.
+        /// </summary>
+        /// <param name="resource">The instance to set the serialization info.</param>
+        /// <param name="serializationInfo">The serialization info to set.</param>
+        public static void SetSerializationInfo(this ODataResource resource, ODataResourceSerializationInfo serializationInfo)
+        {
+            ExceptionUtils.CheckArgumentNotNull(resource, "resource");
+            resource.SerializationInfo = serializationInfo;
+        }
+

```

Comment:
That's merged by other PR, right?

Reply:
Yes; I was working off a branch that already had that commit, but it looks like the merge works fine.

---
In reply to: [162159319](https://github.com/OData/odata.net/pull/1038#discussion_r162159319) [](ancestors = 162159319)